java/plugins/html-elements-plugin/src/test/java/ru/sbtqa/tag/pagefactory/pages/htmlelements/IndexPages/IndexPage.java

```diff
@@ -1,8 +1,8 @@
 package ru.sbtqa.tag.pagefactory.pages.htmlelements.IndexPages;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)